FluentTerminal.SystemTray/Utilities.cs

```diff
@@ -569,44 +569,58 @@ internal static void SaveFile(string path, string content)
             }
         }
 
-        internal static void MuteTerminal(bool mute)
+        private static bool MuteConhost(bool mute, int attemptCount)
         {
-            bool? deviceMuted = VolumeControl.GetDefaultAudioEndpointMute();
-            if (deviceMuted != true)
-            {
-                VolumeControl.SetDefaultAudioEndpointMute(true);
-            }
-
-            Process cmdProcess = new Process();
-            cmdProcess.StartInfo.FileName = "cmd.exe";
-            cmdProcess.StartInfo.WindowStyle = ProcessWindowStyle.Hidden;
-            cmdProcess.StartInfo.Arguments = "/k \x07";
-            cmdProcess.Start();
-
-            bool wasConhostMuted = false;
+            bool audioSessionFound = false;
             int attempt = 0;
-            while (wasConhostMuted == false && attempt++ < 2 && !cmdProcess.HasExited)
+            while (audioSessionFound == false && attempt++ < attemptCount)
             {
-                cmdProcess.WaitForExit(1500);
-
                 foreach (Process conhost in Process.GetProcessesByName("conhost"))
                 {
                     try
                     {
-                        bool? isMuted = null;
-                        isMuted = VolumeControl.GetAudioSessionMute(conhost.Id);
-                        if (isMuted != null && isMuted != mute)
+                        bool? isAudioSessionMuted = null;
+                        isAudioSessionMuted = VolumeControl.GetAudioSessionMute(conhost.Id);
+                        if (isAudioSessionMuted != null && isAudioSessionMuted != mute)

```

Comment:
There's no need for the null check. C# will do the right thing here if this is just:
```suggestion
                        if (isAudioSessionMuted != mute)
```

Reply:
`null` there means that audio session is not found for the process and we shouldn't call `SetAudioSessionMute` for it.

I haven't checked yet, but `if (isAudioSessionMuted != mute)` seems will result in `true` every time when `isAudioSessionMuted` is null and it's not what expected because of above.

Reply:
Sorry, you're right. My suggestion would only work for `==`, not `!=`. Ignore this!